docker/Dockerfile.dev

```
@@ -1,13 +1,15 @@
 FROM esphome/esphome-base-amd64:3.0.0
```

Huh, weird that this container is using `esphome-base` as the base image - we also have [`esphome/esphome-lint`](https://hub.docker.com/r/esphome/esphome-lint) that already ships the `clang-format/tidy/etc` packages (built here: [Dockerfile1](https://github.com/esphome/esphome/blob/dev/docker/Dockerfile.lint), [Dockerfile2](https://github.com/esphome/esphome-docker-base/blob/master/Dockerfile.lint)).

Maybe there's a reason for using that base image, but can you check if it works by just changing this line to
```suggestion
FROM esphome/esphome-lint:latest
```

Then the devcontainer start process would also be much faster, as the packages don't need to be installed first.

---

Thanks. I will check it